lib/lago/__init__.py

```diff
@@ -620,8 +643,8 @@ def _ova_to_spec(self, filename):
 
         if image_file is not None:
             disk_meta = {"root-partition": "/dev/sda1"}
-
-            disk_spec = [{"type": "file",
+            disk_spec = [{"type": "template",
+                          "template_type": "file",
```

Comment:
this doesn't change anything from the outside world.
neither the user see any change in behaviour.
this file is a temporary file used by lago.
I can even delete it and the user will / should not be aware of it.